azure-pipelines.yml

```diff
@@ -62,9 +62,9 @@ stages:
         pool:
           name: $(PoolProvider) # This is a queue-time parameter; Public default is NetCorePublic-Pool; Internal default is NetCoreInternal-Pool
           ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-            demands: ImageOverride -equals Build.Server.Amd64.VS2017.Open
+            demands: ImageOverride windows.vs2019.amd64.open
```

Comment:
Sigh. this was a bug. Due to how I did a draft and then converted to a real PR it didn't actually run. The official build will be fine but I need to fix this line in a followup